api/src/main/java/org/apache/cloudstack/api/command/user/offering/ListDiskOfferingsCmd.java

```diff
@@ -111,14 +109,7 @@ public String getStorageType() {
     }
 
     public State getState() {
-        if (StringUtils.isBlank(diskOfferingState)) {
-            return Active;
-        }
-        State state = EnumUtils.getEnumIgnoreCase(State.class, diskOfferingState);
-        if (!diskOfferingState.equalsIgnoreCase("all") && state == null) {
```

Comment:
Are these checks not relevant anymore?

Reply:
I just made this uniform to how it works with listing network and vpc offerings ; wherein when the state isn't passed, we list all offerings - so we do not explicitly need "all"